package.json

```diff
@@ -41,7 +41,7 @@
     "protobufjs": "^5.0.0",
     "ts-node": "^1.7.0",
     "tslint": "^4.0.0",
-    "typescript": "^2.1.0"
+    "typescript": "2.4.2"
```

Comment:
Although this is a compiler question so it's not actually that critical to do this, I suppose